### DISCUSSION TITLE
Linux deployment issues

### DISCUSSION THREAD

**Discussion by EdnY1** [CefView/QCefView#410] [Q&A] [answered]
hello, I'm planning to use QCefView for a hybrid cross platform application where the UI is gonna be web based, I managed to compile and run on windows successfully, then I cloned source code and compiled successfully on Linux but when I try to run it, I get this error 
![image](https://github.com/CefView/QCefView/assets/79129327/361833fa-e6fa-46ef-8eb5-f5d9005d923c)
I spent 4 hours trying to debug this issue, but no results, am I missing some context? I would appreciate some help.

Here some information about my environment:

- OS: Linux Mint 21.3
- QT: 6.7.1
- Compiler: Clang 17

**Comment by tishion** [comment, marked as answer]
Check your Qt installation on linux, you may forget  to install some qt plugins.

**Reply by EdnY1** [reply]
Thank you, the app was not detecting the plugins path

**Comment by L-Super** [comment]
If you are in the debug env, it is recommended that you set the Qt env to resolve the dependency libraries. If you are in the release step, it is recommended to use linuxdeployqt tool.
